comps/agent/langchain/src/utils.py

```diff
@@ -99,6 +99,19 @@ def setup_llm(args):
     return model
 
 
+def setup_chat_model(args):
```

Comment:
Looks this function is same as setup_vllm_client, can we merge as one?

Reply:
yes. I updated the code.